Documentation/Diagnostics/PH2146.md

```diff
@@ -0,0 +1,104 @@
+# PH2146: Avoid problematic using statement patterns
```

Comment:
Change this analyzer to PH2149. (PH2146 is claimed by another analyzer in a different PR.) Update this file, file name, Diagnostic Id, etc.